bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/SystemProfiles.java

```diff
@@ -32,4 +34,12 @@
             .newTrigger(RAWBUTTON_TOGGLE_SWITCH, "Raw Button Toggle").withSupportedItemTypes(CoreItemFactory.SWITCH)
             .withSupportedChannelTypeUIDs(DefaultSystemChannelTypeProvider.SYSTEM_RAWBUTTON.getUID()).build();
 
-}
+    TriggerProfileType RAWBROCKER_TO_ON_OFF_TYPE = ProfileTypeBuilder
```

Comment:
I assume it should be `RAWROCKER_TO_ON_OFF_TYPE` without the "B" :-)

Reply:
There is no a `RAWBUTTON_TOGGLE_SWITCH` without a "to" between "rawbutton" and "toggle switch".
There is a rawrocker to dimmer that states the type and not the values the conversion is done two (same as toggle switch).
There is a rawrocker to on off that is using the values the conversion is applied to.

Couldn't we make it more consistent and use a "to" for all or for none?
Shouldn't we use "switch" instead of "on off"?

Reply:
Oh, this is a stupid mistake.
@maggu2810 I didn't use switch for the specific reason that this profile can be used for dimmers as well, if you only want to have on and off commands sent to the dimmer. That way it doesn't confuse anyone into thinking it can only be used with switches.
What do you think?